src/Appwrite/Migration/Version/V21.php

```diff
@@ -177,4 +182,20 @@ protected function fixDocument(Document $document): Document
 
         return $document;
     }
+
+    /**
+     * Migrating Buckets.
+     *
+     * @return void
+     * @throws Exception
+     * @throws PDOException
+     */
+    private function migrateBuckets()
+    {
+        foreach ($this->documentsIterator('buckets') as $bucket) {
+            $bucketId = 'bucket_' . $bucket['$internalId'];
+
+            $this->projectDB->updateAttribute($bucketId, 'metadata', size: 75000);
+        }
```

Comment:
We should wrap with try catch so it doesn't fail catastrophically and purge cache. For example: 

https://github.com/appwrite/appwrite/blob/4b1bf59091b747c5855c355b6f48a5346b1cbe0c/src/Appwrite/Migration/Version/V19.php#L101-L114